src/main/java/net/tylers1066/beaming/sign/CrewSign.java

```diff
@@ -24,7 +29,21 @@
 import org.bukkit.event.player.PlayerRespawnEvent;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public class CrewSign implements Listener {
+
+    private final Map<String, Craft> respawnCrafts = new HashMap<>();
+
+    private void removeRespawnCraft(@NotNull Craft craft) {
```

Comment:
Based on the fact that this operation is fairly common, we may want to look into some sort of bidirectional map.

Reply:
FastUtil has a lot of good options for this, but might require that we shade it into Beaming as well.

Reply:
Yeah I agree, but since Beaming depends on the base Movecraft can't we just assume that it has already loaded FastUtil?